otel-playground/otelcol/otelcol-config.yaml

```diff
@@ -0,0 +1,86 @@
+exporters:
+  otlp/sampled:
+    endpoint: "localhost:8080"
+    headers:
+      "x-honeycomb-team": ""
+    compression: zstd
+    sending_queue:
+      enabled: true
+      num_consumers: 2
+  debug:
+    verbosity: detailed
+  otlp:
+    endpoint: "api.honeycomb.io:443"
+    headers:
+      "x-honeycomb-team": ""
```

Comment:
How do you handle these locally, @mterhar ?   would it make sense use an env var here?

Reply:
Since environment stuff goes into the docker-compose.yaml, it'd still end up in the repo. I just hard-code it into the config file so I don't lose track since there are multiple keys at play for the different signals coming out.